src/toolkits/object_detection/one_shot_object_detection/one_shot_object_detector.cpp

```diff
@@ -263,9 +261,9 @@ one_shot_object_detector::one_shot_object_detector() {
 }
 
 gl_sframe one_shot_object_detector::augment(
-    const gl_sframe &data, const std::string &image_column_name,
-    const std::string &target_column_name, const gl_sarray &backgrounds,
-    std::map<std::string, flexible_type> &options) {
+    const gl_sframe& data, const std::string& image_column_name,
+    const std::string& target_column_name, gl_sarray& backgrounds,
```

Comment:
Why do the backgrounds need to be mutable?

Reply:
Fixed.